### PR TITLE
Update getting started steps

### PR DESCRIPTION
**Pull request by F3PiX**
One of the students pointed out that the 'resubmit' info is hard to find, because it is called 'iterations' everywhere except on this page :-)
- Added subtitles for easy scanning
- Updated step 11 with reference to step 8
- Updated steps 11+ 12 to reflect the steps with approval and completion in the new Exercism. ( + 'will happen' -> 'may happen'

### CODE REVIEW

File: pages/getting_started.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 9. In your web-browser, go back to the [language tracks](/tracks) page, choose your exercise and the exercise you've just worked on. You will now see your solution online and notice that it is awaiting a mentor.
 10. A mentor will now come and give you feedback on your solution. This can happen really quickly (15 minutes) or take a little while (a day or two) depending on demand. In the meantime, you will find other exercises on your track are available for you to solve!
-11. Once you've received feedback, you can make some changes to your solution and then resubmit it. This will happen a couple of times before the solution is completed.
-12. Once the solution is completed you will unlock lots more exercises.
+
+###### Iterations, resubmit 
+11. Once you've received feedback, you can make some changes to your solution and then resubmit it (see step 8). This may happen a couple of times before the solution is approved. 
+12. Once the solution is approved by a mentor, you click 'complete' to mark the solution completed. This will unlock lots more exercises
```
Comment (iHiD) [inline pages/getting_started.md]: Mind throwing a full-stop on the end of this pls?

**Comment by iHiD** [comment]
@F3PiX Brilliant. Thanks :) I've added one comment. If you can edit the PR to do that, great. Otherwise I'll sort it after I've merged. I'll give it a couple of hours then merge either way :)

**Comment by F3PiX** [comment]
@ihid Thanks for catching that! Back to you.

**Comment by iHiD** [comment]
Perfect. Thanks :)

**Pull request merged**